tests/simulator/refinement_boxes.py

```diff
@@ -37,13 +38,30 @@ def dup(dic):
     valid1D = [
         dup({"cells":[65], "refinement_boxes": {"L0": {"B0": [(10,), (14,)]}}}),
         dup({"cells":[65], "refinement_boxes": {"L0": {"B0": [(5,), (55,)]}}}),
+        dup({"cells":[65], "refinement_boxes": {"L0": {"B0": Box(5, 55)}}}),
+        dup({"cells":[65], "refinement_boxes": {"L0": [Box(5, 55)]}}),
+        dup({"cells":[65], "refinement_boxes": {  0 : [Box(5, 55)]}}),
+        dup({"cells":[65], "refinement_boxes": {  0 : [Box(0, 55)]}}),
+        dup({"cells":[65], "refinement_boxes": {"L0": [Box(5, 14), Box(15, 25)]}}),
+        dup({"cells":[65], "refinement_boxes": {"L0": [Box(5, 25)], "L1": [Box(12, 48)], "L2": [Box(60, 64)]}}),
+        dup({"cells":[65], "refinement_boxes": {"L0": [Box(5, 25)], "L1": [Box(12, 48)]}, "max_nbr_levels": 3}),
```

Comment:
do we accept max_nbr_levels in cases refinement boxes are supplied?

Reply:
we don't anymore!